ui/oopsyraidsy/data/00-misc/general.ts

```diff
@@ -33,10 +33,12 @@ const raiseAbilityIds = [
 ];
 
 const targetMitAbilityIdToDuration: { [id: string]: number } = {
+  // Casters and melees gain a L98 trait that increases Feint and Addle from 10s to 15s
+  // TODO: Add a level check to determine duration?
   '1D6F': 10, // reprisal
-  '1D7D': 10, // feint
+  '1D7D': 15, // feint
```

Comment:
Reprisal also gets a trait to boost it to 15s.

Reply:
Oops, totally forgot about that...